packages/form/blueprints/form/FormInput.json

```diff
@@ -0,0 +1,24 @@
+{
+  "type": "system/SIMOS/Blueprint",
```

Comment:
protocol missing

same for all other types in the file

Reply:
Does not look resolved.
And think we should try and with "CORE:Blueprint" syntax when we can

Reply:
fixed